executor/check.go

```diff
@@ -48,12 +55,13 @@ func (c *CheckExecutor) Execute() error {
 	}
 	checkJob.GetLabels()[job.K8upExclusive] = "true"
 
-	return c.startCheck(checkObject, checkJob)
+	return c.startCheck(checkJob, checkObject)
 }
 
-func (c *CheckExecutor) startCheck(checkObject *k8upv1alpha1.Check, checkJob *batchv1.Job) error {
+func (c *CheckExecutor) startCheck(checkJob *batchv1.Job, checkObject *k8upv1alpha1.Check) error {
 	c.check = checkObject
```

Comment:
Line 62 assigns does `c.check = checkObject`, why can't we do that within Execute() already? that would save us a parameter